sentinelhub/geometry.py

```diff
@@ -1,9 +1,11 @@
 """
 Module implementing geometry classes
 """
+from __future__ import annotations
```

Comment:
this is just to avoid `"BBox"` and the likes? I don't really mind

Reply:
Yes, it seems nicer to me, but I don't know if it brings any benefit at all.

Reply:
aside from aesthetic ones I don't think so. At least not in this case